modules/administration-guide/partials/proc_setting-up-the-bitbucket-application-link.adoc

```diff
@@ -0,0 +1,100 @@
+:_content-type: PROCEDURE
+:description: Setting up the Bitbucket application link
+:keywords: configuring-authorization, configure-authorization, private-repository, private-git-repository, private-repo, private-git-repo, private-bitbucket, private, bitbucket, bitbucket-repo, bitbucket-repository
+:navtitle: Setting up the Bitbucket application link
+// :page-aliases:
+
+[id="setting-up-the-bitbucket-application-link_{context}"]
+= Setting up the Bitbucket application link
+
+Set up a Bitbucket application link using OAuth 1.0 as follows:
+
+.Prerequisites
+
+* You are logged in to Bitbucket.
+* link:https://www.openssl.org/[`openssl`] is installed in the operating system you are using.
+* link:https://www.gnu.org/software/coreutils/base64[`base64`] is installed in the operating system you are using.
+
+.Procedure
+
+. On a command line, generate the `privatepkcs8-stripped.pem` key in the required format and with Base64 encoding:
++
+[source,subs="+quotes,+attributes,+macros"]
+----
+$ openssl genrsa -out private.pem 2048
+
+$ openssl pkcs8 -topk8 -inform pem -outform pem -nocrypt -in private.pem -out privatepkcs8.pem
+
+$ cat privatepkcs8.pem | sed 's/-----BEGIN PRIVATE KEY-----//g' | sed 's/-----END PRIVATE KEY-----//g' | tr -d '\n' | base64 | tr -d '\n' > privatepkcs8-stripped.pem
+----
+
+. Save the Base64-encoded `privatepkcs8-stripped.pem` file for the next procedure (Applying the Bitbucket application link Secret).
+
+. On a command line, generate the `public-stripped.pub` key in the required format:
++
+[source,subs="+quotes,+attributes,+macros"]
+----
+$ openssl rsa -in private.pem -pubout > public.pub
+
+$ cat public.pub | sed 's/-----BEGIN PUBLIC KEY-----//g' | sed 's/-----END PUBLIC KEY-----//g' | tr -d '\n' > public-stripped.pub
+
+----
+
+. On a command line, generate a Base64-encoded consumer key:
++
+[source,subs="+quotes,+attributes,+macros"]
+----
+$ openssl rand -base64 24 > bitbucket-consumer-key
+----
+
+. Save the Base64-encoded `bitbucket_consumer_key` file for the next procedure (Applying the Bitbucket application link Secret).
```

Comment:
Again, I wouldn't reference other procedures as you don't know what other procedures users will go to or complete. 

How about: `Save the Base64-encoded bitbutcket_consumer_key file.`